packages/base/time.gts

```diff
@@ -0,0 +1,16 @@
+import { Component } from './card-api';
+import StringField from './string';
+import { BoxelInput } from '@cardstack/boxel-ui/components';
+
+export default class TimeCard extends StringField {
+  static displayName = 'Time';
+  static edit = class Edit extends Component<typeof this> {
+    <template>
+      <BoxelInput
+        @value={{@model}}
+        @onInput={{@set}}
+        @type='time'
+      />
+    </template>
+  };
+}
```

Comment:
If you make a package in base. We must be certain it is good for all use-cases and include validation checks. 

My suggestion is simple, have this component exist in the event.gts card for now. Also, it has to extend from FieldDef not StringField (I think its confusing otherwise)